Mathlib/Analysis/NormedSpace/PiLp.lean

```diff
@@ -560,6 +560,25 @@ theorem nnnorm_eq_ciSup {β : ι → Type*} [∀ i, SeminormedAddCommGroup (β i
   simp [NNReal.coe_iSup, norm_eq_ciSup]
 #align pi_Lp.nnnorm_eq_csupr PiLp.nnnorm_eq_ciSup
 
+@[simp] theorem nnnorm_equiv {β : ι → Type*} [∀ i, SeminormedAddCommGroup (β i)] (f : PiLp ∞ β) :
+    ‖WithLp.equiv ⊤ _ f‖₊ = ‖f‖₊ := by
```

Comment:
Mini-section with variables, and then the statements should fit on one line.
```suggestion
section Linfty
variable {β : ι → Type*} [∀ i, SeminormedAddCommGroup (β i)]

@[simp] theorem nnnorm_equiv (f : PiLp ∞ β) : ‖WithLp.equiv ⊤ _ f‖₊ = ‖f‖₊ := by
```

Reply:
Done; it turned out there was already an ambient `β`